### PR TITLE
Features/refactor healthz apis token verifier

### PR DESCRIPTION
**Pull request by mangal-napses**

### CODE REVIEW

File: lib.js [hunk 1]
```diff
@@ -1,32 +1,32 @@
-const {
```
Comment (bhattisatish) [inline lib.js:1]: Why do you prefer the explicit require per function instead of block require?

File: lib.js [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+	logDebug: utilityLogger.logDebug,
 	composeResult,
 	ifElse,
 	respond,
 	transformToResult,
 	whenResult,
 	withArgs,
 	doNothing,
-	uuid,
+	uuid: { ...uuid, v4: () => uuid.v4() },
```
Comment (bhattisatish) [inline lib.js:27]: Can we use uuid v7? Is there a js lib that allows us to generate uuid V7?

File: utils/redis.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    try {
+        let url = `redis://${redis.host}:${redis.port}`;
+        if (redis.username && redis.password) {
+            url = `redis://${redis.username}:${redis.password}@${redis.host}:${redis.port}`;
+        }
+        client = createClient({ url });
+        await client.connect();
+
+        client.on('error', (err) => {
+            console.log(`Error occured during connection with redis. ${err}`);
```
Comment (bhattisatish) [inline utils/redis.js]: This should be logError

File: utils/redis.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+        });
+        client.on('connect', () => {
+            console.log('Connected to redis successfully :)');
+        });
+    } catch (err) {
+        client = {}
+        console.log(`Could not establish a connection with redis. ${err}`);
```
Comment (bhattisatish) [inline utils/redis.js]: logError

**Review by bhattisatish — APPROVED** [review]

File: lib.js (hunk 1 above)
Comment (mangal-napses) [reply lib.js:1]: It was loading 12M of size if we load from utilities, but when i was loading from each one separate, each one was of size around 10-20k only.

File: lib.js (hunk 2 above)
Comment (mangal-napses) [reply lib.js:27]: https://www.npmjs.com/package/uuidv7

we can try this

File: utils/redis.js (hunk 3 above)
Comment (mangal-napses) [reply utils/redis.js]: updated

**Pull request merged**